scripts/!mods_preload/msu_mod_item.nut

```diff
@@ -2,7 +2,39 @@ local gt = this.getroottable();
 
 gt.MSU.modItem <- function ()
 {
-	gt.Const.Items.addNewItemType <- function( _itemType )
+	gt.Const.Items.ItemTypeName <- [];
+	gt.Const.Items.ItemTypeName.resize(gt.Const.Items.ItemType.len(), "");
+	
+	foreach (itemType in this.Const.Items.ItemType)
+	{
+		local name = "";
+		switch (itemType)
+		{
+			case this.Const.Items.ItemType.Legendary:
+				name = "Legendary Item";
+				break;
+
```

Comment:
if we are using this type of `switch` syntax with an empty line, I know some of my code needs to be adjusted so it's all standardized